embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ! pip3 install --upgrade --user langchain==0.1.16 gcsfs==2024.3.1 etils==1.8.0
</pre><p>Could you change the langchain install to just be langchain-core langchain-text-splitters and langchain-google-community?</p><p> </p><p>It'll make it faster to install</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done